web/privacy.html

```diff
@@ -0,0 +1,521 @@
+
+        <p>In addition, the data subject has the right, on grounds relating to his or her particular situation, to object to processing of personal data concerning him or her by the joind.in for scientific or historical research purposes, or for statistical purposes pursuant to Article 89(1) of the GDPR, unless the processing is necessary for the performance of a task carried out for reasons of public interest.</p>
+
+        <p>In order to exercise the right to object, the data subject may contact any employee of the joind.in. In addition, the data subject is free in the context of the use of information society services, and notwithstanding Directive 2002/58/EC, to use his or her right to object by automated means using technical specifications.</p>
+    </li>
+    <li><h4>h) Automated individual decision-making, including profiling</h4>
+        <p>Each data subject shall have the right granted by the European legislator not to be subject to a decision based solely on automated processing, including profiling, which produces legal effects concerning him or her, or similarly significantly affects him or her, as long as the decision (1) is not is necessary for entering into, or the performance of, a contract between the data subject and a data controller, or (2) is not authorised by Union or Member State law to which the controller is subject and which also lays down suitable measures to safeguard the data subject's rights and freedoms and legitimate interests, or (3) is not based on the data subject's explicit consent.</p>
+
+        <p>If the decision (1) is necessary for entering into, or the performance of, a contract between the data subject and a data controller, or (2) it is based on the data subject's explicit consent, the joind.in shall implement suitable measures to safeguard the data subject's rights and freedoms and legitimate interests, at least the right to obtain human intervention on the part of the controller, to express his or her point of view and contest the decision.</p>
+
+        <p>If the data subject wishes to exercise the rights concerning automated individual decision-making, he or she may, at any time, contact any employee of the joind.in.</p>
+
+    </li>
+    <li><h4>i) Right to withdraw data protection consent </h4>
+        <p>Each data subject shall have the right granted by the European legislator to withdraw his or her consent to processing of his or her personal data at any time. </p>
+
+        <p>f the data subject wishes to exercise the right to withdraw the consent, he or she may, at any time, contact any employee of the joind.in.</p>
+
+    </li>
+</ul>
+<h4>9. Data protection provisions about the application and use of Facebook</h4>
+<p>On this website, the controller has integrated components of the enterprise
```

Comment:
We don't have a "share on facebook" button only the possibility to log in via facebook. So this will need to be rewritten…